app/javascript/src/Dashboard/tabs-widget.jsx

```diff
@@ -0,0 +1,19 @@
+const CURRENT_TAB_CLASS = 'current-tab'
+
+export function initialize () {
```

Comment:
Was this approach simpler than recreating the rendered view in React?

Reply:
I think so, the dashboard apis section is rather complex and moving it to React would require extra work. IMO right now this is the quicker and even more efficient way.